extensions/azurecore/extension.webpack.config.js

```diff
@@ -16,6 +16,8 @@ const externals = {
 	'bufferutil': 'commonjs bufferutil',
 	'utf-8-validate': 'commonjs utf-8-validate',
 	'keytar': 'commonjs keytar',
+	'@azure/arm-subscriptions': 'commonjs @azure/arm-subscriptions',
```

Comment:
You've verified that this fixes the issue? I'm surprised - webpacking the arm packages aren't the problem here and so having them be external wouldn't change anything for that I would think.